pkg/model/components/docker.go

```diff
@@ -73,17 +73,9 @@ func (b *DockerOptionsBuilder) BuildOptions(o interface{}) error {
 	docker.IPMasq = fi.Bool(false)
 
 	// Note the alternative syntax... with a comma nodeup will try each of the filesystems in turn
-	if b.IsKubernetesGTE("1.11") {
-		// TODO(justinsb): figure out whether to use overlay2 on AWS jessie:
-		// The ContainerOS image now has docker configured to use overlay2 out-of-the-box
-		// and it is an error to specify the flag twice.
-		// But Jessie (still our default AWS image) isn't recommended by docker with overlay2
-		// (though that may be a kernel issue, and we run a custom kernel on our default image)
-		// But we still need to worry about users running generic AMIs (e.g. stock jessie)
-		docker.Storage = fi.String("overlay2,overlay,aufs")
-	} else {
-		docker.Storage = fi.String("overlay,aufs")
-	}
+	// TODO(justinsb): The ContainerOS image now has docker configured to use overlay2 out-of-the-box
+	// and it is an error to specify the flag twice.
+	docker.Storage = fi.String("overlay2,overlay,aufs")
```

Comment:
@hakman Are overlay and aufs still relevant? (It would be a separate PR if not.)

Reply:
All supported distros should have support for overlay2, so I think you are right and the rest could be removed.

Reply:
Removing this completely is also a possibility, as Docker already has a pretty good list of priorities:
https://github.com/docker/docker-ce/blob/v19.03.11/components/engine/daemon/graphdriver/driver_linux.go#L51

Reply:
I look forward to your future PR.

Reply:
👍